### PR TITLE
added bubble sort in typescript Simpsons rule in c#

### LINKED ISSUE #2143: Bubble Sort
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [x] Dart
- [x] Go
- [x] Java
- [x] Julia
- [x] JS
- [x] Kotlin
- [x] Php
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @m-code12
- [x] Shell
- [ ] TypeScript - @manildayma

### PR DESCRIPTION
**Pull request by manildayma**
Fixes #2143 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- typescript
- 

#### Files Added:
- bubble_sort.ts
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
please remove the irrelevant files

**Comment by manildayma** [comment]
There is just one file @asha15  ma'am. I'm not able to get which file you are talking about.

**Comment by manildayma** [comment]
@asha15  ma'am I request you to review it again.  It's a pull reqy created for two issues by mistake. I hope it can be merged.

**Comment by raghav-dalmia** [comment]
@manildayma remove the `bubble_sort.ts` file.

File: Bubble_Sort/bubble_sort.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+//Bubble Sort In Typescript
```
Comment (raghav-dalmia) [inline Bubble_Sort/bubble_sort.ts]: this one.

**Comment by asha15** [comment]
You was assigned to create bubble sort in type script, but you add a c# code too
Kindly request you to remove it

**Pull request closed without merging**